### DISCUSSION TITLE
QA model

### DISCUSSION THREAD

**Discussion by Marwa199527** [aub-mind/arabert#155] [Q&A] [answered]
Hi @WissamAntoun;
I am using the QA model on a data set that is in SQuAD format, where fine-tuning goes well without any problem but when it starts evaluation this error comes up.

![problem_1](https://user-images.githubusercontent.com/68481906/155567486-f7e6e25f-69cb-4f30-9539-b1c545b94751.PNG)

I used the same code you used for ARCD (in examples file) I only skipped the combination step (my data is large enough and there is no combination needed), I need to make this work as I will use the result in a paper that I am working on.

**Comment by WissamAntoun** [comment]
Hey,

I think this is an issue in the transformers library, i suggest you try using the new `.py` from https://github.com/huggingface/transformers/tree/master/examples/pytorch/question-answering. You will probably need to install the latest version from transformers, also you might face some compatibility issues.

**Comment by WissamAntoun** [comment, marked as answer]
Hey,

I think this is an issue in the transformers library, i suggest you try using the new `.py` from https://github.com/huggingface/transformers/tree/master/examples/pytorch/question-answering. You will probably need to install the latest version from transformers, also you might face some compatibility issues.

**Reply by WissamAntoun** [reply]
you can also try adding these two lines in the utils_qa.py file which might fix the issue.
![image](https://user-images.githubusercontent.com/44616226/155570113-1667472d-aaec-460f-b364-66cfe9326494.png)

**Reply by Marwa199527** [reply]
@WissamAntoun thank you very much it works, keep the hard work, and I just want to say that I enjoyed your lecture about the AraBERT model.
